src/libraries/Common/tests/WasmTestRunner/WasmTestRunner.cs

```diff
@@ -20,24 +16,19 @@ public static async Task<int> Main(string[] args)
     {
         int index = 0;
 
+        var runner = new WasmTestRunner();
 #if SINGLE_FILE_TEST_RUNNER
         // This runner is also used for NativeAOT testing, which defines SINGLE_FILE_TEST_RUNNER.
-        var testAssembly = typeof(SimpleWasmTestRunner).Assembly.GetName().Name;
+        runner.TestAssembly = typeof(WasmTestRunner).Assembly.GetName().Name;
 #else
         if (args.Length == 0)
         {
             Console.WriteLine($"No args given");
             return -1;
         }
 
-<<<<<<< HEAD
-        var testAssembly = args[index++];
-#endif
-=======
-        var runner = new WasmTestRunner();
-
         runner.TestAssembly = args[0];
```

Comment:
```suggestion
        runner.TestAssembly = args[index++];
```
Otherwise the parsing below for `!SINGLE_FILE_TEST_RUNNER` will break.

Reply:
Thanks